pom.xml

```diff
@@ -461,6 +461,7 @@
                                 <exclude>**/*.MF</exclude>
                                 <exclude>**/*.crc</exclude>
                                 <exclude>**/*.parquet</exclude>
+                                <exclude>**/example*.xml</exclude>
```

Comment:
There are some `*.xml` files in `testutils`:
- `testutils/src/test/resources/xml_examples/example1.xml`
- `testutils/src/test/resources/xml_examples/example2.xml`

These shouldn't need a license header, but we don't want to be too permissive and exclude all `*.xml` files or we might end up with `pom.xml` without a header.